commands/core/config.drush.inc

```diff
@@ -462,7 +462,7 @@ function _drush_config_export($destination, $destination_dir, $branch) {
     }
     $comment .= "\n\n$output";
 
-    if (!$commit && !drush_confirm(dt('The .yml files in your export directory (!target) will be deleted.', array('!target' => $destination_dir)))) {
+    if (!$commit && !drush_confirm(dt('The .yml files in your export directory (!target) will be deleted and overwridden with the active config.', array('!target' => $destination_dir)))) {
```

Comment:
I think we need the prompt:
I you're using GIT, yes it is save to reexport, but if you don't track the config files at all, it is not save to export and not prompt the user, as he would loose a config. 
Also we use `drush cex` to shortly have a preview of what would be exported and if this is not exactly what we expect (let's say because there is some configuration that should not be exported), you can still abort right there.